### ISSUE TITLE
Fix creation of test users for selenium testing

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#8871]
### Description
Selenium tests launcher are creating test users before starting tests execution. There are [changes in Eclipse Che deployment](https://github.com/eclipse/che/pull/8819) which should be reflected in the launcher.
Another issue is **admin** user should be mapped to role "read-token" to be able to execute Github-related tests.
